### PR TITLE
feat: add pod security standards (baseline) in CEL expressions - Part 1

### PR DESCRIPTION
**Pull request by MariamFahmy98**
## Related Issue(s)
#773 

## Description
This PR adds Kyverno policies written in CEL expressions for pod security standards (baseline). 
note: I will distribute the work on multiple PRs to make the review process a little bit easier.

## Checklist

- [x] I have read the [policy contribution guidelines](https://github.com/kyverno/policies/blob/main/README.md#contribution).
- [x] I have added test manifests and resources covering both positive and negative tests that prove this policy works as intended.
- [x] I have added the artifacthub-pkg.yml file and have verified it is complete and correct.

### CODE REVIEW

File: pod-security-cel/baseline/disallow-capabilities/disallow-capabilities.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: disallow-capabilities
+  annotations:
+    policies.kyverno.io/title: Disallow Capabilities in CEL expressions
+    policies.kyverno.io/category: Pod Security Standards (Baseline) in CEL
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/minversion: 1.11.0
+    kyverno/kubernetesVersion: "1.22-1.23"
```
Comment (chipzoller) [inline pod-security-cel/baseline/disallow-capabilities/disallow-capabilities.yaml:10]: This annotation key isn't right.

File: pod-security-cel/baseline/disallow-capabilities/disallow-capabilities.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/description: >-
+      Adding capabilities beyond those listed in the policy must be disallowed.
+spec:
+  validationFailureAction: audit
```
Comment (chipzoller) [inline pod-security-cel/baseline/disallow-capabilities/disallow-capabilities.yaml:15]: Need to be compliant with all the latest language so `Audit` throughout.

File: pod-security-cel/baseline/disallow-capabilities/disallow-capabilities.yaml (hunk 1 above)
Comment (chipzoller) [reply pod-security-cel/baseline/disallow-capabilities/disallow-capabilities.yaml:10]: It's right for Artifact Hub but internally we use a different form.

**Review by chipzoller — CHANGES_REQUESTED** [review]
The `kyverno.io/kubernetes-version` version for these state 1.22-1.23. As you know, this is the version of Kubernetes on which the policies were tested and certified to work. Except these versions are rather old at this point. Can you please retest on something like 1.26-1.27?

File: pod-security-cel/baseline/disallow-capabilities/disallow-capabilities.yaml (hunk 2 above)
Comment (MariamFahmy98) [reply pod-security-cel/baseline/disallow-capabilities/disallow-capabilities.yaml]: Done.

File: pod-security-cel/baseline/disallow-capabilities/disallow-capabilities.yaml (hunk 1 above)
Comment (MariamFahmy98) [reply pod-security-cel/baseline/disallow-capabilities/disallow-capabilities.yaml]: I fixed it. Thank you.

**Comment by chipzoller** [comment]
Per Slack discussion, if `validate.message` does not work currently for CEL expression Kyverno policies, the field should be removed.

**Review by chipzoller — APPROVED** [review]
Thank you for the great work!

**Pull request merged**